### ISSUE TITLE
Version 0.8.0 doubles counter when set to track "unread" label

### ISSUE THREAD

**Issue by blacklion** [inbasic/ignotifier#480]
When notifier is configured to track "unread" label (instead of default "inbox" one), and there are unread messages in inbox itself, it doubles counter: one message is shown as two, two as four, etc.
It is new for 0.8.0
Screenshot is attached (I have 2 unread messages in inbox itself and nothing else in my multitude of labels).
![gmail-double-counter](https://user-images.githubusercontent.com/252120/31863504-d810e436-b756-11e7-8f94-cd3b7b525b73.png)

**Comment by inbasic** [comment]
Are you sure there is no label "inbox" for your account as well?
It should be like
![screen shot 2017-10-23 at 3 09 46 pm](https://user-images.githubusercontent.com/351062/31887204-52ef7e58-b804-11e7-809c-f8c591196b67.png)

**Comment by blacklion** [comment]
I have default config with first "inbox" replaced with "unread":
![gmail-notifier-settings-1](https://user-images.githubusercontent.com/252120/31887561-7d861724-b801-11e7-8923-4e3ead822874.PNG)


But still:
![gmail-notifier-couinters-2](https://user-images.githubusercontent.com/252120/31887519-57ec07da-b801-11e7-983a-8a403cc158a1.PNG)

**Comment by blacklion** [comment]
Ooops, I've click on each input field in settings to make sure they are empty and press "Del" several times, and after that is shows only "[unread]" line in popup. Looks like it is glitch of importing old settings?

When Extension was updated yo 0.8.0, it lost all my settings, and I've typed "unread" again instead of "inbox".

So, it is fixed now, but looks like that extension should clean-up old settings if it can not import them.
